js/src/offcanvas.js

```diff
@@ -108,7 +108,7 @@ class Offcanvas extends BaseComponent {
       new ScrollBarHelper().hide()
     }
 
-    this._element.setAttribute('aria-modal', true)
+    this._element.setAttribute('aria-modal', this._config.backdrop !== false)
```

Comment:
```suggestion
    this._element.setAttribute('aria-modal', Boolean(this._config.backdrop))
```

I suppose it is the same

Reply:
Yes, it should be better.
Thanks!